tools/ModpackUploader.ps1

```diff
@@ -48,54 +50,125 @@ if ($IsLinux) {
 
     #Lets Check if the user has Curl Installed
     if (-not (test-path "/usr/bin/curl")) { 
-        Write-Host "Curl needed to use the ModpackUploader." 
+        Write-Host "Curl needed to use the ModpackUploader."
         #If Program is NOT installed stop the script
         return
     }
     Set-Alias cl "curl"
 
 }
-elseif ($IsWindows) {
+elseif ($IsMacOS) {
+
     #Lets Check if the user has 7-Zip Installed
-    if (-not (test-path "$env:ProgramFiles\7-Zip\7z.exe")) {
+    if (-not (test-path "/usr/local/bin/7z")) { 
         Write-Host "7-Zip needed to use the ModpackUploader."
         #If Program is NOT installed stop the script
-        return 
+        return
     }
-    Set-Alias sz "$env:ProgramFiles\7-Zip\7z.exe"
+    Set-Alias sz "7z"
 
     #Lets Check if the user has Curl Installed
-    if (-not (test-path "$env:C:\Windows\System32\curl.exe")) {
+    if (-not (test-path "/usr/bin/curl")) { 
         Write-Host "Curl needed to use the ModpackUploader."
         #If Program is NOT installed stop the script
+        return
+    }
+    Set-Alias cl "curl"
+
+}
+elseif ($IsWindows) {
+    if (test-path "$env:ProgramFiles\7-Zip\7z.exe") {
+        Set-Alias sz "$env:ProgramFiles\7-Zip\7z.exe"
+    }
+    elseif (test-path "$env:USERPROFILE/scoop/apps/7zip/current/7z.exe") {
+        Set-Alias sz "$env:USERPROFILE/scoop/apps/7zip/current/7z.exe"
+    }
+    else {
+        Write-Host "7-Zip needed to use the ModpackUploader."
+        #If Program is NOT installed stop the script
+        return 
+    }
+
+    #Lets Check if the user has Curl Installed
+    if (-not (test-path "C:\Windows\System32\curl.exe")) {
+        Set-Alias cl "C:\Windows\System32\curl.exe"
+    }
+    elseif (test-path "$env:USERPROFILE/scoop/apps/curl/current/bin/curl.exe") {
+        Set-Alias cl "$env:USERPROFILE/scoop/apps/curl/current/bin/curl.exe"
+    }
+    else {
+        Write-Host "cURL needed to use the ModpackUploader."
+        #If Program is NOT installed stop the script
         return 
     }
-    Set-Alias cl "$env:C:\Windows\System32\curl.exe"
 }
 
-if ($ENABLE_MANIFEST_BUILDER_MODULE) {
+if ($ENABLE_CURSE_CLIENT_MODULE) {
```

Comment:
Did you forget to rename this in `settings.ps1`? :)

Reply:
Wow...